server/graphql/v2/mutation/OrderMutations.js

```diff
@@ -0,0 +1,124 @@
+import { GraphQLNonNull } from 'graphql';
+
+import activities from '../../../constants/activities';
+import status from '../../../constants/order_status';
+import models from '../../../models';
+import { NotFound, Unauthorized } from '../../errors';
+import { getDecodedId } from '../identifiers';
+import { OrderReferenceInput } from '../input/OrderReferenceInput';
+import { Order } from '../object/Order';
+
+const modelArray = [
+  { model: models.Subscription },
+  { model: models.Collective, as: 'collective' },
+  { model: models.Collective, as: 'fromCollective' },
+];
+
+const orderMutations = {
+  cancelOrder: {
+    type: Order,
+    description: 'Cancel an order',
+    args: {
+      input: {
```

Comment:
This argument is usually named after the object type.
```suggestion
      order: {
```

Reply:
Thanks for that! I didn't know. Will update.